VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -30,13 +29,12 @@
     @ColorInt
     private final int edx_utility_success_dark;
 
-    @Inject
-    public DiscussionPostsAdapter(Context context, IEdxEnvironment environment) {
-        super(context, R.layout.row_discussion_thread, environment);
-        edx_brand_primary_base = context.getResources().getColor(R.color.edx_brand_primary_base);
-        edx_grayscale_neutral_light = context.getResources().getColor(R.color.edx_grayscale_neutral_light);
-        edx_brand_secondary_dark = context.getResources().getColor(R.color.edx_brand_secondary_dark);
-        edx_utility_success_dark = context.getResources().getColor(R.color.edx_utility_success_dark);
+    public DiscussionPostsAdapter(Fragment fragment, IEdxEnvironment environment) {
+        super(fragment.getContext(), R.layout.row_discussion_thread, environment);
```

Comment:
Yes, you're most likely correct about that. Most of the view injections don't need to be defined as a local variable at all, since they're not referenced after initialization. However, this is out of scope here, and can be handled in a separate ticket/PR (or the whole system will be replaced eventually when we start applying MVP everywhere).